vllm/v1/worker/tpu_input_batch.py

```diff
@@ -0,0 +1,609 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I suppose this is the old gpu_input_batch for compatibility until we figure TPU out right?